### PR TITLE
Internationalization: Mark up PlaylistEditPage for translations 

### LINKED ISSUE #74366: i18n: Mark up `PlaylistEditPage` for translations
follow instructions [here](https://github.com/grafana/grafana/blob/main/contribute/internationalization.md#how-to-add-a-new-translation-phrase). can verify translation strings have been marked up correctly by using the development-only pseudo locale: 
image

checklist:
- [ ] Page title
- [ ] Page description
- [ ] Form field labels
- [ ] `Dashboards` heading
- [ ] `Dashboards` empty state text
- [ ] `Add dashboards` heading
- [ ] `Delete playlist item` button tooltip text
- [ ] `Select dashboard` placeholder text
- [ ] `Select a tag` placeholder text
- [ ] `Save button`
- [ ] `Cancel button`

### PR DESCRIPTION
**Pull request by khushijain21**
**What is this feature?**

i18n markup for Playlist edit page

**Which issue(s) does this PR fix?**:
This PR fixes https://github.com/grafana/grafana/issues/74366

### CODE REVIEW

**Comment by khushijain21** [comment]
Hello, this is my first PR in i18n, let me know if any corrections required

File: public/app/features/playlist/PlaylistEditPage.tsx [hunk 1]
```diff
@@ -25,17 +26,26 @@ export const PlaylistEditPage = ({ match }: Props) => {
   };
 
   const pageNav: NavModelItem = {
-    text: 'Edit playlist',
-    subTitle:
-      'A playlist rotates through a pre-selected list of dashboards. A playlist can be a great way to build situational awareness, or just show off your metrics to your team or visitors.',
+    text: t('playlist-edit.text', 'Edit playlist'),
```
Comment (ashharrison90) [inline public/app/features/playlist/PlaylistEditPage.tsx:29]:
```suggestion
    text: t('playlist-edit.title', 'Edit playlist'),
```

File: public/app/features/playlist/PlaylistEditPage.tsx [hunk 2]
```diff
@@ -25,17 +26,26 @@ export const PlaylistEditPage = ({ match }: Props) => {
...
+        {playlist.error && (
+          <Trans i18nKey="playlist-edit.error">
+            <div> Error loading playlist: {JSON.stringify(playlist.error)}</div>
+          </Trans>
+        )}
+        {playlist.value && (
+          <Trans i18nKey="playlist-edit.value">
+            <PlaylistForm onSubmit={onSubmit} playlist={playlist.value} />
+          </Trans>
+        )}
```
Comment (ashharrison90) [inline public/app/features/playlist/PlaylistEditPage.tsx:48]: we don't want to wrap the whole `PlaylistForm` in `Trans`, instead we need to go into this file and wrap the individual text items. you can refer to the [checklist in the issue](https://github.com/grafana/grafana/issues/74366) to find which text items need translating

**Review by ashharrison90 — CHANGES_REQUESTED** [review]
hi @khushijain21 👋 

great start! left some comments to nudge you in the right direction 😉

File: public/app/features/playlist/PlaylistEditPage.tsx [hunk 3]
```diff
@@ -25,16 +26,22 @@ export const PlaylistEditPage = ({ match }: Props) => {
   };
 
   const pageNav: NavModelItem = {
-    text: 'Edit playlist',
-    subTitle:
-      'A playlist rotates through a pre-selected list of dashboards. A playlist can be a great way to build situational awareness, or just show off your metrics to your team or visitors.',
+    text: t('playlist-edit.title', 'Edit playlist'),
+    subTitle: t(
+      'playlist-edit.subTitle',
```
Comment (ashharrison90) [inline public/app/features/playlist/PlaylistEditPage.tsx:31]:
```suggestion
      'playlist-edit.sub-title',
```

File: public/app/features/playlist/PlaylistEditPage.tsx [hunk 4]
```diff
@@ -25,16 +26,22 @@ export const PlaylistEditPage = ({ match }: Props) => {
...
   };
 
   return (
     <Page navId="dashboards/playlists" pageNav={pageNav}>
       <Page.Contents isLoading={playlist.loading}>
-        {playlist.error && <div>Error loading playlist: {JSON.stringify(playlist.error)}</div>}
-
+        {playlist.error && (
+          <div>
+            <Trans i18nKey="playlist-edit.errorPrefix">Error loading playlist:</Trans>
```
Comment (ashharrison90) [inline public/app/features/playlist/PlaylistEditPage.tsx:41]:
```suggestion
            <Trans i18nKey="playlist-edit.error-prefix">Error loading playlist:</Trans>
```
(i realise `errorPrefix` was originally my suggestion, apologies for not suggesting in kebab-case)

File: public/app/features/playlist/PlaylistForm.tsx
Comment (ashharrison90) [inline public/app/features/playlist/PlaylistForm.tsx:1]: think we could nest all these under the `playlist-edit` key, so `playlist-form.name-label` would become `playlist-edit.form.name-label`. same comment about making sure these are all in kebab case

**Review by ashharrison90 — COMMENTED** [review]
couple of tweaks, otherwise lgtm 🥳

**Review by ashharrison90 — COMMENTED** [review]
just need to update the `addTitle`/`addTag` keys to be kebab case (`add-title-label`, `add-tag-label`, `add-tag-placeholder`) and rerun `yarn i18n:extract`/`yarn i18n:pseudo` and this will be good to merge 👍

**Comment by ashharrison90** [comment]
@khushijain21 do you need any further guidance in this PR? 🙂

**Review by ashharrison90 — APPROVED** [review]
looks great! thanks for the contribution @khushijain21 🥳

**Pull request merged**